assets/javascripts/application.js

```diff
@@ -167,5 +168,6 @@ $(function() {
   accordion();
   charCounter();
   addRemove();
+  modalDialog().init();
```

Comment:
I'll remove modalDialog.init() from application.js and move the code inline at the end of the modalDialog.js